tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -253,6 +253,12 @@ def create_environment_list(self, test_suites: List[str]) -> List[Dict[str, Any]
 
         environments.extend(shared_environments.values())
 
+        if len(skip_test_suites) > 0:
+            self._log.info("Skipping test suites %s", skip_test_suites)
+
+        if len(skip_test_suites_images) > 0:
+            self._log.info("Skipping test suits run on images \n %s", '\n'.join([f"\t{skip}" for skip in skip_test_suites_images]))
+
```

Comment:
I moved this code here, just before we raise the exception, since these messages can help understand why the exception was raised.